src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/HabitComponent.java

```diff
@@ -19,35 +19,24 @@
 
     private String habitTitle;
 
-    private WebElement habitChart;
-
-    private WebElement moreElementsButton;
-
     private List<WebElement> habitItems;
 
-    private List<WebElement> estimationButtons;
+    private List<WebElement> estimationButtons = new ArrayList<>();
 
     public HabitComponent(WebElement habitLayout) {
         this.habitLayout = habitLayout;
         initElements();
     }
 
     private void initElements() {
-        habitTitle = habitLayout.findElement(By.cssSelector("app-habit-title h4")).getText();
-
-        habitChart = habitLayout.findElement(By.cssSelector("app-habit-chart"));
-
-        moreElementsButton = habitLayout.findElement(By.cssSelector("app-habit-item-list span"));
-
-        estimationButtons = new ArrayList<WebElement>();
-        habitLayout.findElements(By.cssSelector("button")).forEach(item -> estimationButtons.add(item));
+        habitTitle = habitLayout.findElement(By.className("title")).getText();
 
         initHabitItems();
     }
 
     private void initHabitItems() {
         habitItems = new ArrayList<WebElement>();
-        habitLayout.findElements(By.cssSelector("img")).forEach(item ->  habitItems.add(item));
+        habitLayout.findElements(By.cssSelector(".habits-in-progress > .ng-star-inserted")).forEach(item -> habitItems.add(item));
```

Comment:
could you improve this selector with more reliable?

Reply:
Unfortunately no. There no other identification, just CssSelectors.